frontend/talentsearch/src/js/components/search/SearchFilterAdvice.tsx

```diff
@@ -1,61 +1,150 @@
   const intl = useIntl();
   if (
-    classificationFilterCount === 0 &&
-    operationalRequirementFilterCount === 0
+    operationalRequirementFilterCount === 0 &&
+    !educationSelection &&
+    !workingLanguage &&
+    !employmentDuration &&
+    !equityFiltersActive &&
+    skillCount === 0
   ) {
     return null;
   }
 
   const recommendations = [];
-  if (classificationFilterCount > 0) {
+  if (operationalRequirementFilterCount > 0) {
     recommendations.push({
-      key: "classifications",
+      key: "operationalRequirements",
       link: (
         <a
-          href="#classificationsFilter"
+          href="#operationalRequirementFilter"
           data-h2-color="base(dt-primary)"
           data-h2-font-weight="base(700)"
         >
           {intl.formatMessage(
             {
               defaultMessage:
-                "Classification Filters ({classificationFilterCount})",
-              id: "LRysDa",
+                "Conditions of Employment ({operationalRequirementFilterCount})",
+              id: "ky585k",
             },
-            { classificationFilterCount },
+            { operationalRequirementFilterCount },
           )}
         </a>
       ),
     });
   }
-  if (operationalRequirementFilterCount > 0) {
+
+  if (educationSelection) {
     recommendations.push({
-      key: "operationalRequirements",
+      key: "educationRequirementFilter",
       link: (
         <a
-          href="#operationalRequirementFilter"
+          href="#educationRequirementFilter"
+          data-h2-color="base(dt-primary)"
+          data-h2-font-weight="base(700)"
+        >
+          {intl.formatMessage({
+            defaultMessage: "Diploma required",
+            id: "MG4+wL",
+          })}
+        </a>
+      ),
+    });
+  }
+
+  if (workingLanguage) {
+    recommendations.push({
+      key: "workingLanguageFilter",
+      link: (
+        <a
+          href="#workingLanguageFilter"
+          data-h2-color="base(dt-primary)"
+          data-h2-font-weight="base(700)"
+        >
+          {intl.formatMessage({
+            defaultMessage: "Language ability selection",
+            id: "BV+Zp4",
+          })}
+        </a>
+      ),
+    });
+  }
+
+  if (employmentDuration) {
+    recommendations.push({
+      key: "employmentDurationFilter",
+      link: (
+        <a
+          href="#employmentDurationFilter"
+          data-h2-color="base(dt-primary)"
+          data-h2-font-weight="base(700)"
+        >
+          {intl.formatMessage({
+            defaultMessage: "Duration selection",
+            id: "5F6Cky",
+          })}
+        </a>
+      ),
+    });
+  }
+
+  if (equityFiltersActive) {
+    recommendations.push({
+      key: "employmentEquityFilter",
+      link: (
+        <a
+          href="#employmentEquityFilter"
+          data-h2-color="base(dt-primary)"
+          data-h2-font-weight="base(700)"
+        >
+          {intl.formatMessage({
+            defaultMessage: "Equity selection",
```

Comment:
It might be nice to add the number of filters in parenthesis the way CoE and Skills do it.

Reply:
sure https://github.com/GCTC-NTGC/gc-digital-talent/pull/4553/commits/f21f9af77d266d060eb9940a45fb5429428d7dc6